swap/swap.go

```diff
@@ -95,7 +133,7 @@ func new(stateStore state.Store, prvkey *ecdsa.PrivateKey, backend contract.Back
 }
 
 // New - swap constructor with integrity checks
-func New(dbPath string, prvkey *ecdsa.PrivateKey, backendURL string, disconnectThreshold uint64, paymentThreshold uint64) (*Swap, error) {
+func New(logpath string, dbPath string, prvkey *ecdsa.PrivateKey, backendURL string, disconnectThreshold uint64, paymentThreshold uint64) (*Swap, error) {
```

Comment:
place the `logpath` argument in the same order as it is used in the function body

Reply:
I not sure i understood
could you elaborate a bit more?.

Reply:
i think it's a good idea to define parameters in the `New` function in the same order as they are used in its body. but it's not a major gripe